snakefmt/formatter.py

```diff
@@ -85,13 +84,11 @@ def read_black_config(self, path: PathLike) -> black.FileMode:
                 continue
 
             snakecase_config[key] = val
+        if "line_length" not in snakecase_config:
+            snakecase_config["line_length"] = DEFAULT_LINE_LENGTH
 
         return black.FileMode(**snakecase_config)
 
-    @property
-    def line_length(self) -> int:
```

Comment:
Why did you get rid of this?

Reply:
What purpose was it serving in your opinion? From what I could tell, it was not serving a specific purpose, we only used it as a possibly distinct `line_length` passed to black. Now I propose using a single `line_length`, so this is no longer useful. My opinion is we can always reintroduce it at a later time if we find we want to use `line_length` in a context other than passing it to black

Reply:
I agree it was serving a seemingly useless purpose originally, but if we change the behaviour of how we interact with line length, that behaviour can be handled by this function without needing to changed our interaction with line length throughout the codebase.
Would it be useful to have some of the interaction with `DEFAULT_LINE_LENGTH` contained within here?

Reply:
`DEFAULT_LINE_LENGTH` is only used once now, at `__init__`, so i don't see the point of having that function personally.